src/cobalt-host.h

```diff
@@ -15,11 +16,15 @@ const char *cobalt_host_get_app_id(CobaltHost *host, GError **error);
 
 const char *cobalt_host_get_app_exec(CobaltHost *host, GError **error);
 
+SemVer *cobalt_host_get_fp_version(CobaltHost *host, GError **error);
```

Comment:
Also a bit of a nit: this can be private to cobalt-host (at least for now), like `check_for_binary` and such are.